arrays.py

```diff
@@ -598,4 +613,37 @@
     "a Protogen",
     "a cutie",
     "a pizza",
+    "some Python 2 code",
+    "some Java 7 code",
+    "another jq implementation",
+    "a pair of programmer socks",
+    "an LED strip",
+    "a LED strip",
```

Comment:
Purposeful almost-duplicate to the above?

Reply:
yes, I wrote “an” first, then wondered if there’s people who would use “a” and whether they might complain about the bot using “an”, checked enwiktionary (“/ɛl.iːˈdiː/, /lɛd/”), and decided it would be funny to include both

we can call it a social experiment and see which posts get more replies about the article ;P

Reply:
I have never heard anyone pronounce it as a single syllable before and I never want to